### PR TITLE
November 2024 rke2 v2.10

### PR DESCRIPTION
**Pull request by briandowns**

### CODE REVIEW

File: channels-rke2.yaml [hunk 1]
```diff
@@ -2404,6 +2404,14 @@ releases:
         repo: rancher-rke2-charts
         version: 1.9.001
     featureVersions: *featureVersions-v1
+  - version: v1.29.11-rc2+rke2r1
+    minChannelServerVersion: v2.9.0-alpha1
+    maxChannelServerVersion: v2.9.99
```
Comment (ShylajaDevadiga) [inline channels-rke2.yaml]: maxChannelServerVersion needs to be v2.10.99 right?

**Comment by VestigeJ** [comment]
@briandowns this looks like it still needs both
[v1.30.7-rc3+rke2r1](https://github.com/rancher/rke2/releases/tag/v1.30.7-rc3%2Brke2r1)
[v1.29.11-rc2+rke2r1](https://github.com/rancher/rke2/releases/tag/v1.29.11-rc2%2Brke2r1)
Screenshot 2024-12-05 at 9 37 00 AM

**Review by kinarashah — APPROVED** [review]

File: channels-rke2.yaml (hunk 1 above)
Comment (VestigeJ) [reply channels-rke2.yaml]: Could this prevent the previous RCs from showing correctly ?

Screenshot 2024-12-05 at 5 09 31 PM

Comment (kinarashah) [reply channels-rke2.yaml]: Yeah, I might’ve missed this while reviewing versions yesterday. For 2.10, max rancher version should be 2.10.99

**Review by ShylajaDevadiga — APPROVED** [review]

**Review by kinarashah — APPROVED** [review]

**Pull request merged**